docx.gemspec

```diff
@@ -11,6 +11,7 @@ Gem::Specification.new do |s|
   s.email       = ['chrahunt@gmail.com']
   s.homepage    = 'https://github.com/chrahunt/docx'
   s.files       = Dir["README.md", "LICENSE.md", "lib/**/*.rb"]
+  s.required_ruby_version = '>= 2.4.6'
```

Comment:
Please specify only 0 as micro version, like so: `>= 2.4.0`. This prevents issues for people using Rubies installed by OS package managers (rpm, deb, etc).

Reply:
Thanks @mvz ! I fixed.